site/en/blog/new-in-chrome-89/index.md

```diff
@@ -0,0 +1,223 @@
+---
+title: New in Chrome 89
+description: >
+  Chrome 89 is rolling out now! WebHID, WebNFC, and Web Serial have graduated
+  from their origin trials and are now available in stable. We're closing a
+  loophole a few developers used to skirt the PWA installability checks.
+  Web Share, and Web Share Target arrive on the desktop. And there's plenty
+  more!
+layout: 'layouts/blog-post.njk'
+date: 2021-03-02
+authors:
+  - petelepage
+hero: 'image/0g2WvpbGRGdVs0aAPc6ObG7gkud2/st3dNRCY5FpDmmZXJ8ZG.jpg'
+alt: >
+  New in Chrome hero logo
+tags:
+  - new-in-chrome
+  - chrome-89
+---
+
+Chrome 89 is starting to roll out to stable now.
+
+{% YouTube id='O-gv6zLZHKw' %}
+
+Here's what you need to know:
+
+* [WebHID, WebNFC, and Web Serial](#hardware) have graduated from their
+  origin trials and are now available in stable.
+* We're closing a loophole a few developers used to skirt the
+  [PWA installability checks](#installability).
+* [Web Share, and Web Share Target](#webshare) arrive on the desktop.
+* And, there's [plenty more](#more).
+
+I'm [Pete LePage](https://twitter.com/petele), working, and shooting
+from home, let's dive in and see what's new for developers in Chrome 89!
+
+## WebHID, WebNFC, and Web Serial {: #hardware }
+
+I'm really excited about WebHID, WebNFC, and Web Serial. They open up new
+scenarios for users that were never possible before, interacting with real
+world hardware.
+
+It allows makers to connect to fun, quirky hardware, video conferencing apps
```

Comment:
```suggestion
They allows makers to connect to fun, quirky hardware, video conferencing apps
```

Reply:
I realize this is what you said in the video, but referring to a list of features with a singular pronoun has a high chance causing confusion.